Formula/bluepill.rb

```diff
@@ -0,0 +1,15 @@
+class Bluepill < Formula
+  desc "Bluepill is a reliable iOS testing tool that runs UI tests using multiple simulators on a single machine"
```

Comment:
Description needs to be limited to 80 characters. You may leave out the obvious stuff, e.g.,
"iOS testing tool that runs UI tests using multiple simulators on a machine" or
"Reliable iOS UI testing tool that can run multiple simulators on a machine".